packages/yourloops/pages/patient/teams/add-dialog.tsx

```diff
@@ -257,30 +258,31 @@ export function ConfirmTeam(props: ConfirmTeamProps): JSX.Element {
       </DialogTitle>
 
       <DialogContent>
-        <DialogContentText id="team-add-dialog-confirm-info" color="textPrimary">
+        <DialogContentText id="team-add-dialog-confirm-info" color="textPrimary" data-testid='patient-share-data'>
           {t('modal-patient-add-team-info')}
         </DialogContentText>
-        <DialogContentText id="team-add-dialog-confirm-team-infos" color="textPrimary">
+        <DialogContentText id="team-add-dialog-confirm-team-infos" color="textPrimary" data-testid="patient-team-info">
           {props.team.name}
           <br />
           {teamAddress}
           {props.team.code}
         </DialogContentText>
 
-        <DialogContentText id="team-add-dialog-confirm-team-warning" color="textPrimary">
+        <DialogContentText id="team-add-dialog-confirm-team-warning" color="textPrimary" >
```

Comment:
the id could be removed
```suggestion
        <DialogContentText color="textPrimary">
```